tool/dev.sh

```diff
@@ -4,5 +4,5 @@ cd "$(dirname "$0")/.."
 
 ./tool/build-dev-container.sh
 
-echo "Running development instance on http://localhost. To access it in an Android Emulator use http://10.0.2.2"
+echo "Running development instance on http://localhost. To access it in an Android Emulator use http://10.0.2.2 and on IOS emulator use http://localhost"
```

Comment:
I think this is redundant as the message mentions `localhost` and only explicitly mentions `10.0.2.2` for an android emulator.

Please revert that